### ISSUE TITLE
Can't feed any animal

### ISSUE THREAD

**Issue by FTL-Space-Warp** [TerraFirmaCraft/TerraFirmaCraft#1744]
**Describe the bug**

When attempting to feed animals, nothing happens. I've tried right-clicking normally and while shifted, I've tried all types of grains (the ones refined with a knife) in both creative and survival mode. Horses and llamas make an angry sound, while pigs and cow do not react in any way. Trying to feed wolves with meat or bones has the same result.
The empty heart correctly displays on top of the animal when shifted.
The config file `animal_food_data.json` seems to be correct, it contains:

``` 
"tfc:alpacatfc": {
    "eat_rotten": true,
    "foods": {
      "grain": {
        "type": "forge:ore_dict",
        "ore": "grain"
      }
    }
  },
```
for every animal except wolves

**Meta Info**
 - TFC Version: 1.7.17.175
 - the issue persists with no other mods installed

**Comment by Charnuz** [comment]
Could you upload the entire json file please?

**Comment by FTL-Space-Warp** [comment]
It suddenly started working and at first I had no idea why, but I realized it most likely only starts working after the second day. I only tested it in brand new worlds so I kept thinking it was bugged until a day passed and I tried it again (animals spawned later still work even if they haven't been alive for a day). The same happened to a friend of mine on a different pc. At this point I don't think the json file is related but I could send it anyway if you want it.
